packages/pnpm/src/cmd/index.ts

```diff
@@ -1,16 +1,15 @@
+import * as audit from '@pnpm/cli-audit'
+import { list, why } from '@pnpm/cli-list-and-why'
+import * as outdated from '@pnpm/cli-outdated'
+import { pack, publish } from '@pnpm/cli-pack-and-publish'
```

Comment:
Maybe&nbsp;you&nbsp;could change&nbsp;this, so&nbsp;that you&nbsp;do:

```ts
import { audit } from '@pnpm/plugin-commands-audit'
import { list, why } from '@pnpm/plugin-commands-list-and-why'
import { outdated } from '@pnpm/plugin-commands-outdated'
import { pack, publish } from '@pnpm/plugin-commands-pack-and-publish'
```

or&nbsp;perhaps:
```ts
import audit from '@pnpm/plugin-commands-audit'
import { list, why } from '@pnpm/plugin-commands-list-and-why'
import outdated from '@pnpm/plugin-commands-outdated'
import { pack, publish } from '@pnpm/plugin-commands-pack-and-publish'
```

by&nbsp;introducing&nbsp;`index.ts` into&nbsp;`plugin‑commands‑audit` and&nbsp;`plugin‑commands‑outdated` that&nbsp;simply&nbsp;re‑exports the&nbsp;module&nbsp;namespace&nbsp;object of&nbsp;the&nbsp;actual CLI&nbsp;command implementation&nbsp;module as&nbsp;a&nbsp;named&nbsp;export.

Reply:
I was planning to the first one but using package names that don't contain `-and-`

```ts
import { audit } from '@pnpm/plugin-commands-audit'
import { list, why } from '@pnpm/plugin-commands-listing'
import { outdated } from '@pnpm/plugin-commands-outdated'
import { pack, publish } from '@pnpm/plugin-commands-publishing'
```